integration_tests/features/Sync.feature

```diff
@@ -103,10 +103,10 @@ Feature: Block Sync
     When I mine 15 blocks on PNODE2
     Then all nodes are at height 23
 
-  @broken
   Scenario: Node should not sync from pruned node
-    Given I have a base node NODE1 connected to all seed nodes
-    Given I have a pruned node PNODE1 connected to node NODE1 with pruning horizon set to 5
+    Given I do not expect all automated transactions to succeed
+    And I have a base node NODE1 connected to all seed nodes
+    And I have a pruned node PNODE1 connected to node NODE1 with pruning horizon set to 5
```

Comment:
Look at what I did here: you can turn it on and off again. It can be on here, and only turned off later in the test, and turned on again later